<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BIDDS Release Data</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>patch.ico</normaloff>patch.ico</iconset>
  </property>
  <widget class="QLineEdit" name="line_BEName">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>BE Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_BELat">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>N/S DD MM.MMMM</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Latitude</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_BELong">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>E/W DDD MM.MMMM</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>18</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Longitude</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Input Bullseye Data&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>601</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>341</width>
     <height>191</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>version 0.5</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>40</y>
     <width>241</width>
     <height>191</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Create Debrief Card</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_TXTLoad">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>Copys FDR and DRD File to BIDDS Folder, Open BIDDS</string>
    </property>
    <property name="text">
     <string>Select TXT File</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_CS">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Callsign</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbGPS">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>GPS File</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>10</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Lobo</string>
   </property>
  </widget>
  <zorder>frame_2</zorder>
  <zorder>frame</zorder>
  <zorder>line_BEName</zorder>
  <zorder>label_6</zorder>
  <zorder>line_BELat</zorder>
  <zorder>label_7</zorder>
  <zorder>line_BELong</zorder>
  <zorder>label_8</zorder>
  <zorder>label_9</zorder>
  <zorder>progressBar</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
 </widget>
 <tabstops>
  <tabstop>line_BEName</tabstop>
  <tabstop>line_BELat</tabstop>
  <tabstop>line_BELong</tabstop>
  <tabstop>line_CS</tabstop>
  <tabstop>pushButton_TXTLoad</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
